src/components/job.js

```diff
@@ -273,36 +237,33 @@ class Job extends React.Component {
         },
       },
       {
+        
         title: 'Call API',
         render: () => {
-          return(<div><p>
-              Now that the Job contract has been funded you are able to call the API on the Agent. Select a file to be analyzed by dragging and dropping the file onto the upload
-              area or by clicking the upload area to initiate the file-chooser dialog. Once you have chosen a file to analyze, click the "Call Agent API" button to initate the API call. This
-              will prompt one further interaction with MetaMask to sign your API request before submitting the request to the Agent. This interaction does not initiate a transaction
-              or transfer any additional funds.</p>
-
-              {
-                !this.state.fileUploaded &&
-                <React.Fragment>
-                    <br/>
-                    <br/>
-                    <Upload.Dragger name="file" accept=".jpg,.jpeg,.png" beforeUpload={(file)=>{ this.processFile(file); return false; }} >
-                        <p className="ant-upload-drag-icon">
-                            <Icon type="inbox" />
-                        </p>
-                        <p className="ant-upload-text">Click for file-chooser dialog or drag a file to this area to be analyzed.</p>
-                    </Upload.Dragger>
-                </React.Fragment>
-            }
-
-            <br/>
-            <br/>
-            <Button type="primary" onClick={() => {this.showModal(serviceModal); this.callApi()}} disabled={!this.state.fileUploaded} >Call Agent API</Button>
-          </div>)
-        },
+          return(
+            <p>
+            Now that the Job contract has been funded you are able to call the API on the Agent. Select a file to be analyzed by dragging and dropping the file onto the upload
+            area or by clicking the upload area to initiate the file-chooser dialog. Once you have chosen a file to analyze, click the "Call Agent API" button to initate the API call. This
+            will prompt one further interaction with MetaMask to sign your API request before submitting the request to the Agent. This interaction does not initiate a transaction
+            or transfer any additional funds.
+            </p>
+            )
+        }
       },
+      {
+        
+        title: 'Done',
+        render: () => {
+          return(
+            <p>
+            Your request has been completed.
+            </p>
+            )
+        }
+      }
     ];
-
+    const CallComponent = this.props.callComponent;
+    let skipSNet = false; // for debugging service components
```

Comment:
We should definitely avoid having testing stuff be part of the production codebase